Source/JavaScriptCore/llint/LowLevelInterpreter.cpp

```diff
@@ -587,13 +587,23 @@ JSValue CLoop::execute(OpcodeID entryOpcodeID, void* executableAddress, VM* vm,
 #define OFFLINE_ASM_ALIGN4B ".balign 4\n"
 #define OFFLINE_ASM_NOALIGN ""
 
+#if CPU(ARM64) || CPU(ARM64E)
+#define OFFLINE_ASM_ALIGN_TRAP(align) ".balignl " #align ", 0xd4388e20\n"
+#elif CPU(X86_64)
+#define OFFLINE_ASM_ALIGN_TRAP(align) ".balign " #align ", 0xcc\n"
+#elif CPU(ARM)
+#define OFFLINE_ASM_ALIGN_TRAP(align) ".balignw " #align ", 0xde00\n"
+#endif
```

Comment:
When moving this here, deleted the comments explaining the magic numbers, which are apparently `brk`, `int 3`, and `udf`. I think it would be a good idea to retain those helpful comments in some form.

Reply:
Good catch, I agree! Should be addressed in https://github.com/WebKit/WebKit/commit/80e767de49617fc794cf5da181661202586b711f